packages/react/src/components/dropzone.tsx

```diff
@@ -57,17 +57,17 @@ export function UploadDropzone<TRouter extends FileRouter>(
   const $props = props as unknown as UploadDropzoneProps<TRouter> & {
     // props not exposed on public type
     // Allow to set internal state for testing
-    __internal_state?: "readying" | "ready" | "uploading",
+    __internal_state?: "readying" | "ready" | "uploading";
```

Comment:
how did this work before? 🤔

Reply:
__internal_XXX are properties we added in the styling PR to be able to control the component state without user interaction to show the different states in the documentation

Reply:
No, I mean the commas in a type def. AFAIK that should throw terrible errors, no?

Reply:
Not sure about the errors you are mentioning. Worked fine in this case

Reply:
No, both commas and semicolons are valid in type objects

Reply:
Huh, TIL